pkg/scheduler/framework/types.go

```diff
@@ -406,6 +413,9 @@ type PodInfo struct {
 	RequiredAntiAffinityTerms  []AffinityTerm
 	PreferredAffinityTerms     []WeightedAffinityTerm
 	PreferredAntiAffinityTerms []WeightedAffinityTerm
+	// cachedResource shouldn't be used directly.
+	// Use calculateResource method to obtain the podResource.
```

Comment:
nit: probably it's nicer to describe briefly why we can cache this (esp a reason around InPlacePodVerticalScaling) so that implementation readers wouldn't get a doubt for it.